.template/addons/phrase/.phrase.yml

```
@@ -0,0 +1,18 @@
+phrase:
+  project_id: [PROJECT_ID]
+  file_format: yml
+  push:
+    sources:
+      - file: ./config/locales/<tag>.<locale_name>.yml
+        params:
+          skip_upload_tags: true
+  pull:
+    targets:
+      - file: ./config/locales/<tag>.<locale_name>.yml
+        params:
+          include_empty_translations: true
+          tags: "en"
```

```suggestion
      - file: ./config/locales/<locale_name>.yml
        params:
          include_empty_translations: true
```

For the root translation files, shouldn't it be like this? 🤔